Helper/Activity.php

```diff
@@ -19,14 +19,13 @@ public function createActivity($storeId, $type)
         $endPoint = $this->dataHelper->getActivityEndpoint();
         $client   = $this->apiClient->getClient($storeId);
         
-        $data = array(
-            'type'          => $type,
-            'project_token' => $token,
-            'signature'     => md5($token . $type . $secret)
-        );
+        $data = [
+            'type'   => $type,
+            'secret' => $secret
+        ];
         
-        $url = $endPoint . '/tracking/' . $token . '/activity';
+        $url = $endPoint . '/tracking/' . $token . '/activity'; // should be modified once the enpoint is ready
```

Comment:
What does the comment mean?

Reply:
As far as i know there is a scheduled task to create new activity endpoint that can handle the new hashing algorithm and I was not sure if the url is going to stay the same.

Reply:
I see. The endpoint won't change, we made it so it works with both types of auth on the same endpoint

Reply:
Ok, removing it.